examples/access-control-migration/src/__tests__/acceptance/project.acceptance.ts

```diff
@@ -78,8 +78,8 @@ describe('AccessControlApplication - permissions', () => {
   /**
    * Test a role's permission when visit the 5 endpoints in the
    * project controller
-   * @param role
-   * @param permissions
+   * @param role - role
```

Comment:
I find these comments as not very useful. Describing a `role` parameter as "role" is not adding any new information for readers.

Why do we need this change? Is tsdoc linter complaining about params with no description? If that's true, then can we simply remove the `@param role` line entirely?

Maybe it's a good thing that the linter is complaining about missing description and we should use this as an opportunity to improve our API docs?

Reply:
This was a quick dirty fix. I was trying to see what else is left to be fixed.